### PR TITLE
Update make file to detect OS

### PR DESCRIPTION
**Pull request by Victoriam7**

### CODE REVIEW

**Comment by egc2846** [comment]
works on ubuntu :+1:

**Review by keenanm500 — APPROVED** [review]

**Review by egc2846 — CHANGES_REQUESTED** [review]
``` 
make: 
	python.exe -m PyInstaller $(BUILD_ARGS) src/BeyondRGB.py

clean:
	del .\BeyondRGB.spec
	rmdir /S /Q .\dist
	rmdir /S /Q .\build
	rmdir /S /Q .\out
```
if you could throw this in for managing this on windows and then we can some more testing

**Review by egc2846 — APPROVED** [review]

File: backend/Makefile [hunk 1]
```diff
@@ -12,24 +12,37 @@
...
-make: unix
-
-unix:
-	pyinstaller $(BUILD_ARGS_UNIX) src/BeyondRGB.py
-
-win:
-	pyinstaller $(BUILD_ARGS_WIN) src/BeyondRGB.py
+make: 
+ifeq ($(OS),Windows_NT)  # Windows_NT, XP, 2000, 7, 8, 10, 11...
+	python.exe -m PyInstaller $(BUILD_ARGS) src/BeyondRGB.py
```
Comment (egc2846) [inline backend/Makefile:27]: for some unknown reason when i downloaded this to test it, it only worked with ```pyinstaller $(BUILD_ARGS) src/BeyondRGB.py``` so i guess you can dump the conditional here. other people testing on windows would be great

**Review by egc2846 — CHANGES_REQUESTED** [review]

File: backend/Makefile (hunk 1 above)
Comment (Victoriam7) [reply backend/Makefile:27]: Lets wait to get other tests before changing. We can get these at the meeting tommorow

**Review by egc2846 — APPROVED** [review]

**Pull request merged**